MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+recursive-include openfisca_web_api/i18n *.mo
+include *.md
+include LICENSE
+include *.ini
```

Comment:
@MattiSG Sorry for the late answer.

It's ok with latest setuptools (I'm using 34.3.0 according to pip) and Openfisca git master.

No need to merge the PR.

Thank you !